### PR TITLE
Merge Contact Page Initial Structure & Content

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR is intending to merge the basic structure for the content page into master.
- **Update HTML Structure for Contact Page and Add basic content**
- **Update styling to reflect layout of target design as shown in the image below**
image

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,16 +1,42 @@
...
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: [indent 2 spaces at a time](https://google.github.io/styleguide/htmlcssguide.html#Indentation)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,16 +1,42 @@
...
+		</div>
+		<div class="section">
+		</div>
+        <div class="section">
+		</div>
+        <div class="section" id="contact">
+            <h1>Check me out!</h1>
+            <div class="socials-grid">
+                <a href="">
+                    <img src="https://via.placeholder.com/150/">
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: Tip for later, not for this PR: [provide alternative fallback for multimedia content](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
+body{
+    margin:0;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: update the css similarly to the suggestions in https://github.com/samuelsogbesan/step/pull/7/files/a37e16b4f33284cca5f9c0ce40d45b209e79ae64#diff-680ee8528519da343f72308f826ee0e1

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/index.html]: Indentation fixed in commit 
609c261

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: - Updated tab spacing to +2 in commit 609c261 & 337f88f
-  Add spacing between '{' and selector in 396d0c4
- Added single line break between rules in a05d4c8

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -7,10 +7,29 @@
...
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section" id="contact">
+        <h1>Check me out!</h1>
+        <div class="socials-grid">
+          <a href="">
+          <img src="https://via.placeholder.com/150/">
+          <span>/samuelsogbesan</span>
+          </a href="">
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: don't add attributes to closing tags.

Comment (samuelsogbesan) [reply portfolio/src/main/webapp/index.html]: Updated with commit faae13e

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.section {
+  min-height: 100vh;
+  max-height: 100vh;
+  overflow: hidden;
+  min-width: 100%;
+  max-width: 100%;
+  text-align: center;
+  display: flex;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css:32]: [alphabetize declarations](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order), here and elsewhere

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -7,10 +7,29 @@
...
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section" id="contact">
+        <h1>Check me out!</h1>
+        <div class="socials-grid">
+          <a href="">
+            <img src="https://via.placeholder.com/150/">
+            <span>/samuelsogbesan</span>
+          </a>
+          <a>
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: add an `href` attribute

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -7,10 +7,29 @@
...
-      <p>Click here to get a random message:</p>
+    <div id="fullpage">
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section" id="contact">
+        <h1>Check me out!</h1>
+        <div class="socials-grid">
+          <a href="">
+            <img src="https://via.placeholder.com/150/">
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: [add multimedia fallbacks](
https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)
_(For images whose alt attributes would introduce redundancy, and for images whose purpose is purely decorative which you cannot immediately use CSS for, use no alternative text, as in alt="".)_

**Review by krasimirgg — CHANGES_REQUESTED** [review]
there are still a couple of unresolved comments

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css:32]: Updated in commit e83f090

File: portfolio/src/main/webapp/index.html (hunk 7 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/index.html]: Added multimedia 63747f1

File: portfolio/src/main/webapp/index.html (hunk 6 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/index.html]: Added in commit 9a9abeb

**Comment by samuelsogbesan** [comment]
> there are still a couple of unresolved comments

Resolved few more changes with some fresh commits!

**Review by krasimirgg — APPROVED** [review]
Thank you! Looking good!

**Pull request merged**